repos/system_upgrade/el7toel8/libraries/config/tests/test_version.py

```diff
@@ -74,7 +60,7 @@ def test_matches_version_pass():
     (False, ['7.5']),
 ])
 def test_matches_source_version(monkeypatch, result, version_list):
-    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked())
+    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(src_ver='7.6'))
```

Comment:
It looks like an antipattern (https://en.wikipedia.org/wiki/Magic_number_%28programming%29#Unnamed_numerical_constants). However not sure if it has the sense to solve since it is everywhere now.

Reply:
I believe that this is ok.